software/winrm/src/main/java/org/apache/brooklyn/util/core/internal/winrm/WinRmTool.java

```diff
@@ -86,6 +86,8 @@
             "Size of file chunks (in bytes) to be used when copying a file to the remote server", 
             1024);
 
+    ConfigKey<String> ADDITIONAL_CONNECTION_METADATA = newStringConfigKey("additional.connection.metadata", "Can be used by a location customizer during provisioning to provide additional information, which in turn can be consumed by a custom ssh tool");
```

Comment:
Personal preference for splitting onto multiple lines when it's this crazy-long a line. Otherwise, it's hard to tell if there's a default value or anything like that lurking at the very end. I'd go for one line per arg.